raiden/src/utils/types.ts

```diff
@@ -192,3 +181,9 @@ set(sizedCodecCache, ['20', 'Address'], Address); // register on sized cache, fo
 
 export const Byte = UInt(1);
 export type Byte = t.TypeOf<typeof Byte>;
+
+export const UInt64 = UInt(8);
```

Comment:
The `UInt` factory function is cached, you don't need to define and export the trivial types. Please, remove these exports  and use direct call (`UInt(8)`) and type (`UInt<8>`) where they're needed. Declaring the types is only useful when you want to _semantically_ make a point of the layout of a given type, like `Address` being 20 bytes.

Reply:
My reasoning was mostly for consistency with the rest of the code. If there is no benefit I will remove it no worries.

Reply:
I think it's important to think how **semantically** a given type will pass a new idea. E.g. a `Hash` is well known, and people will understand what guarantees are involved there (in this case, only that it's 32 bytes long, and as so, at compile time is translated directly as `string & HexStringB<32>`, being equivalent to other 32bytes strings).
Maybe e.g. a `Nonce` type would make more sense, so we can specify in a centralized point all the type guarantees, but I don't think the size-only justify specifying a new type. And anyway, `UInt256` and `UInt(256)` have the same semantic value, so it doesn't justify this new type. Instead, using the `UInt` directly in the data model definitions pass exactly the idea we want, and have the same meaning at both compile and runtime.